workos/resources/organizations.py

```diff
@@ -1,29 +1,27 @@
-from workos.resources.base import WorkOSBaseResource
+from typing import List, Literal, Optional, TypedDict
+from workos.resources.workos_model import WorkOSModel
 
 
-class WorkOSOrganization(WorkOSBaseResource):
-    """Representation of WorkOS Organization as returned by WorkOS through the Organizations feature.
+class OrganizationDomain(WorkOSModel):
+    id: str
+    organization_id: str
+    object: Literal["organization"]
```

Comment:
Should this be `Literal["organization_domain"]`?

Reply:
It sure should